vsc/model/vsc_generator.py

```diff
@@ -58,7 +58,7 @@ def get_template(filename):
     return jinja_env.get_template(filename)
 
 # gen() function to be called from templates
-def gen(node : AST, template_file = None):
+def gen(node : Namespace, template_file = None):
```

Comment:
1. The idea to have the type as AST came from the original implementation where all node-types were sub-classes inheriting from the AST class.  Thus the type of node "is-a" AST.  But that class hierarchy does not exist anymore.  

2. Even if the top-level node will be Namespace, the purpose of the gen() function is that it can accept **any** node type.

So... in this version where the class AST does not exist, I would guess that it is just as good to not give a type hint at all.

Reply:
1. Once you mentioned that you wanted to make everything according to the specification and in the specification, there is no root element except `Namespace`. No problem, I will fix it later today and bring `AST` back.

2. Hm, for this purpose in python there is a generic type, something like that would be 
```python
def gen(node: Generic[T], template_file = None):

```

Reply:
@gunnarx done, you can check.

Reply:
well, just thought

```python
def gen(node: Any, template_file = None):
...
```
would work as well

Reply:
> OK... point 1 was just information, not really a change request. As I said the class hierarchy does not exist now, so therefore there is (was) no AST parent type.

I know. I was not sure completely, waiting for your feedback.